luthier/src/main/java/com/yahoo/bard/webservice/config/luthier/FactoryPark.java

```diff
@@ -14,12 +14,14 @@
  */
 public class FactoryPark<T> {
 
-    public static final String FACTORY_KEY = "type";
+    static final String ENTITY_TYPE = "factory park";
```

Comment:
this probably shouldn't be called an 'entity type'.
Other places in our design 'Entity' is an instance of a 'Concept'.